README.md

```diff
@@ -79,3 +79,29 @@ yarn test:e2e
 ```bash
 yarn test
 ```
+
+## Open Dollar App - Docker Deployment Guide
+
+This guide walks you through the steps to run the Open Dollar App using Docker Compose based on the provided docker-compose.yml and Dockerfile.
+##### Pre-requisites:
+Ensure Docker and Docker Compose are installed on your system.
+##### Dockerfile Overview:
+Our Dockerfile is a multi-stage build file with two main stages - 'base' and 'runner'.
+
+* In the 'base' stage, we initialize a Node.js environment. We copy our package.json, yarn.lock, .yarnrc.yml, and .yarn files into the image for yarn package manager configuration. Afterwards, we install our dependencies and copy our source code (public/ and src/) into the image. We then run yarn build to create a production build of our application.
+* In the 'runner' stage, we start with an Apache HTTP Server in Alpine Linux as our base image. Next, we copy the build from 'base' stage to /var/www/html directory in our image.
+##### Docker Compose Overview:
+Our docker-compose.yml file defines our services, networks and volumes. Currently, it only has one service 'app'.
+* 'app': This service builds a Docker image named 'open-dollar-app:latest' using the context as the current directory and Dockerfile for building instructions. It then runs the container, with port 80 of the container mapped to port 8080 of the host.
+##### Running the Services:
+To run the services, navigate to the directory containing the docker-compose.yml file and execute the following command:
+```bash
+docker-compose up --build
```

Comment:
I was not able to view the app after running this command. I only see "It Works" but not the actual app